docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -102,6 +102,7 @@ dependencies:
     - watchdog=2.1
     - websockets
     - yapf=0.40.1
+    - databricks-connect
```

Comment:
If your installing this as a pip package, it needs to be underneath the `- pip:` line. 
If it is available as a conda package, then it should sorted alphabetially.

Reply:
yes, the changes are in my local will be pushed by EOD for the same

Reply:
changes have been made